input/pages/codings.xml

```diff
@@ -124,42 +124,42 @@
   </tr>
   <tr>
     <td>dbSNP<br/>(DBSNP : Single Nucleotide Polymorphism database )</td>
-    <td>The Short Genetic Variations database (dbSNP) is a public-domain archive maintained by NCBI for a broad collection of short genetic polymorphisms. The SNP ID is unique for each position and length of DNA change. For example, a change of 3 nucleotides willhave a different SNP ID than a change of 4 nucleotides at the same locus, but the code will be the same for all changes at the same locus and with the same length. So to specify a variation in relation to a referece sequence, both the alt allele and the SNP code must be included.</td>
+    <td>The Short Genetic Variations database (dbSNP) is a public-domain archive maintained by NCBI for a broad collection of short genetic polymorphisms. The SNP ID is unique for each position and length of DNA change. For example, a change of 3 nucleotides will have a different SNP ID than a change of 4 nucleotides at the same locus, but the code will be the same for all changes at the same locus and with the same length. So, to specify a variation in relation to a reference sequence, both the alt allele and the SNP code must be included.</td>
```

Comment:
Actually, should we spell out '3' and '4' here?

Reply:
sure